frontend/server/libs/dao/Identities.dao.php

```diff
@@ -173,6 +173,33 @@ public static function isVerified($identity_id) {
         return $rs;
     }
 
+    public static function isUserAssociatedWithIdentityOfGroup($userId, $identityId) {
```

Comment:
```suggestion
    public static function isUserAssociatedWithIdentityOfGroup(int $userId, int $identityId) {
```

hay que acostumbrarnos a usar tipos siempre. posiblemente necesites hacer cast donde lo mandas llamar.

Reply:
listo